README.md

```diff
@@ -391,6 +391,35 @@ For more examples and ideas, visit:
 
 ```
 
+#### Proxy Setup
+
+> [!NOTE]
+> **From AXIS OS 12.0**
+>
+> To use the shell script provided to set proxy, [developer mode](#developermode) certificate has to be set on camera, to be able use acap-dockerdwrapper ssh user instead of root ssh user.
```

Comment:
SSH as root requires another (Axis internal) certificate, so rephrase it something like: "... developer mode certificate has to be installed on the camera (device? which term is used), in order to use SSH."

Will the script take care of which username to use? I suggest that it defaults to acap-dockerdwrapper and only uses root if a special option "--root" or similar is passed in.